### PR TITLE
[GLT-3996] added documentation for assay tests

### PR DESCRIPTION
**Pull request by djcooke**
Jira ticket or GitHub issue: https://jira.oicr.on.ca/browse/GLT-3996

- [ ] Includes a change file (n/a)

### CODE REVIEW

File: docs/user_manual/type_data.md [hunk 1]
```diff
@@ -715,6 +715,28 @@ is only deletable if it is not being used by any contacts.
 
 
 
+## Assay Tests
+
+[Assays](../requisitions#assays) detail the work intended to be performed for a requisition. This work is broken down
+into one or more tests. A test describes a workflow of extraction, library preparation, library qualification, and
```
Comment (callunity) [inline docs/user_manual/type_data.md:721]: "workflow of X, Y, and Z" out of curiosity, are tests able to be created with only a subset of these workflow steps? (e.g. a test where we'd expect to receive libraries and perform only library qualification and full-depth sequencing) If so, perhaps `and/or` at the end here?

**Review by callunity — APPROVED** [review]

File: docs/user_manual/type_data.md (hunk 1 above)
Comment (djcooke) [reply docs/user_manual/type_data.md:721]: Not really - the test describes all of these steps, whether they were performed internally or externally before receipt

File: docs/user_manual/type_data.md [hunk 2]
```diff
@@ -715,6 +715,28 @@ is only deletable if it is not being used by any contacts.
...
+full-depth sequencing. For example, an assay may involve whole genome sequencing of tumour and normal/reference
+samples. These would be the assay tests - tumour whole genome, and normal whole genome. The same test may be used for
+multiple assays.
+
+The "repeat per timepoint" field indicates whether the test should be repeated for each timepoint received, or a single
+test is sifficient. There is normally no need to repeat sequencing of a normal/reference sample, so this field would be
```
Comment (avarsava) [inline docs/user_manual/type_data.md:727]: sifficient -> sufficient

Comment (djcooke) [reply docs/user_manual/type_data.md]: Fixed. Thanks!

**Review by avarsava — APPROVED** [review]

**Pull request merged**